opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-taxii-domain.ts

```diff
@@ -47,8 +48,18 @@ export const addIngestion = async (context: AuthContext, user: AuthUser, input:
   return element;
 };
 
-export const patchTaxiiIngestion = async (context: AuthContext, user: AuthUser, id: string, patch: object) => {
-  const patched = await patchAttribute(context, user, id, ENTITY_TYPE_INGESTION_TAXII, patch);
+export interface TaxiiIngestionPatch {
+  current_state_cursor?: string | undefined,
+  last_execution_date?: string,
+  added_after_start?: Moment,
```

Comment:
Not sure why but in ingestionManager it's a Moment.

Reply:
you're right, we use a Moment object when updating the date, but in BasicStoreEntityIngestionTaxii, added_after_start is a Date

Reply:
so you think it should be a Date here ?